src/main/kotlin/com/terraformation/backend/email/EmailService.kt

```diff
@@ -104,6 +104,15 @@ class EmailService(
     }
   }
 
+  /**
+   * Sends an email notification to a support email if configured.
+   *
+   * @param [model] Model object containing values that can be referenced by the template.
+   */
+  fun sendSupportNotification(model: EmailTemplateModel) {
+    config.support.email?.let { Locale.ENGLISH.use { send(model, listOf(it)) } }
```

Comment:
The use of `it` is slightly unclear here since it's in a nested lambda. Maybe introduce a name for it like `email`?